contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/test_meta_rename.py

```diff
@@ -33,18 +32,16 @@ def setUp(self):
 
     self.new_name = 'goo'
     self.spec_path = 'a'
-    self.set_options(**{ 'from': '{}:a'.format(self.spec_path), 'to': '{}:{}'.format(self.spec_path, self.new_name) })
-    self.meta_rename = self.create_task(self.context(target_roots=prepare_dependencies(self).values()))
+    self.set_options(**{ 'from': '{}:a'.format(self.spec_path),
```

Comment:
(Unrelated to your change, but I'd like to learn...) This seems like a weird use of kwargs... Is there a reason this isn't just:

```
self.set_options(
  from='{}:a'.format(self.spec_path),
  to='{}:{}'.format(self.spec_path, self.new_name),
)
```

?

Reply:
I'm assuming it's because `from` is a keyword, so python chokes on using it as a kwarg name. It was not a good idea to have an option of this name...